Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("EventDispatcher") {
+
+            var eventDispatcher: EventDispatcher!
+            var callbackWasCalled: Bool!
+
+            let eventName = "some-event"
+            let callback: EventCallback = { _ in
+                callbackWasCalled = true
+            }
+
+            beforeEach {
+                eventDispatcher = EventDispatcher()
+                callbackWasCalled = false
+            }
+
+            describe("on") {
+                it("executes callback function") {
+                    eventDispatcher.on(eventName, callback: callback)
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == true
+                }
+
+                it("receives userInfo when triggered with params") {
```

Comment:
This ``it`` seems that is holding a context logic, do you think that is necessary to split into a ``context`` > ``it`` ?
It could be:
```swift
context("#on") {
    context("When triggered with params") {
        it("receives userInfo") {}
    }
}
```